python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -26,3 +26,40 @@ def fit(m, x, y=None):
 
 def fit_transform(m, x):
     m.fit_transform(x)
+
+
+def predict(m, x, y=None):
+    m.predict(x) if y is None else m.predict(x, y)
+
+
+def _fil_classification_setup(m, data, arg={}):
+    from cuml.utils.import_utils import has_xgboost
+    if has_xgboost():
+        import xgboost as xgb
+    else:
+        raise ImportError("No XGBoost package found which is required for benchmarking FIL")
+    import os 
+
+    tmp_path = "./fil_models"
+    if not os.path.exists(tmp_path):
+        os.makedirs(tmp_path)
+
+    # use maximum 1e6 rows to train the model 
+    train_size = min(data[0].shape[0], 1000000)
+    dtrain = xgb.DMatrix(data[0][:train_size, :], label=data[1][:train_size])
+    params = {"silent": 1, "eval_metric": "error", "objective": "binary:logistic"}
+    params.update(arg)
+
+    max_depth = arg["max_depth"]
+    num_rounds = arg["num_rounds"]
+    n_features = data[0].shape[1]
+    model_name = "xgb_" + str(max_depth) + "_" + str(num_rounds) + "_" + str(n_features) + "_" + str(train_size) + ".model"
```

Comment:
would be more pythonic as `f"xgb_{max_depth}_{num_rounds}_{n_features}_{train_size}.model"`

Reply:
Thanks!